src/qt/region.cpp

```diff
@@ -111,15 +111,17 @@ wxRegion::wxRegion(const wxBitmap& bmp, const wxColour& transp, int tolerance)
 
     QImage img(bmp.GetHandle()->toImage());
     int r = transp.Red(), g = transp.Green(), b = transp.Blue();
-    for(int y=0; y<img.height(); y++) {
-        for(int x=0; x<img.width(); x++) {
+    for(int y=0; y<img.height(); y++)
```

Comment:
the same spacing that applies to `if ( condition )` should be applied to `for ( init; cond; incr )`.
also, `y=0;` should be `y = 0;`